### PR TITLE
[201811] Dell S6100:  Modify transceiver change event from interrupt to poll mode

### PR DESCRIPTION
**Pull request by aravindmani-1**
#### Why I did it

- xcvrd crash was seen in latest 201811 images.
- For Dell S6100,API 2.0 uses poll mode while 1.0 was still using interrupt mode.

#### How I did it

- Modified get_transceiver_change_event in 1.0 to poll mode in all the related branches.

#### How to verify it

- Do OIR testing, check xcvrd status and syslogs.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

#### Description for the changelog
UT logs:
[201811_s6100_change_event_UT.txt](https://github.com/Azure/sonic-buildimage/files/6315166/201811_s6100_change_event_UT.txt)

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh [hunk 1]
```diff
@@ -37,7 +37,6 @@ remove_python_api_package() {
 if [[ "$1" == "init" ]]; then
     modprobe i2c-dev
     modprobe i2c-mux-pca954x force_deselect_on_exit=1
-    modprobe dell_ich
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh:40]: As per https://github.com/Azure/sonic-buildimage/pull/7336, should you revert the changes to this file?

**Review by jleveque — CHANGES_REQUESTED** [review]

File: platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh (hunk 1 above)
Comment (aravindmani-1) [reply platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh:40]: No harm in removing this change in 201811. Will commit the new changes.

**Review by jleveque — APPROVED** [review]
